src/tess_asteroids/movingtpf.py

```diff
@@ -387,7 +391,9 @@ def create_aperture(self, method: str = "threshold", **kwargs):
         if method == "threshold":
             self.aperture_mask = self._create_threshold_mask(**kwargs)
         # will add these methods in a future PR
-        elif method in ["prf", "ellipse"]:
+        elif method == "ellipse":
+            self.aperture_mask = self._ellipse_aperture(ellipse_params=False, **kwargs)
```

Comment:
Could we rename function `_create_ellipse_aperture` to be consistent with threshold aperture?